packages/runtime-core/src/componentRenderContext.ts

```diff
@@ -8,12 +8,19 @@ import { closeBlock, openBlock } from './vnode'
  */
 export let currentRenderingInstance: ComponentInternalInstance | null = null
 export let currentScopeId: string | null = null
-
-export function setCurrentRenderingInstance(
-  instance: ComponentInternalInstance | null
-) {
+const currentRenderingInstanceStack: ComponentInternalInstance[] = []
```

Comment:
does this work without exporting the variable like `currentRenderingInstance` and `currentScopeId`?

Reply:
No, they still need to be exported.

Reply:
so shouldn't `currentRenderingInstanceStack` be exported as well? Or is it because it's a `const`?